### PR TITLE
Read repo

### PR DESCRIPTION
**Pull request by marinakr**

### CODE REVIEW

File: apps/core/config/config.exs [hunk 1]
```diff
@@ -8,9 +8,10 @@ use Mix.Config
 # General application configuration
 config :core,
   namespace: Core,
-  ecto_repos: [Core.Repo, Core.DeduplicationRepo],
+  ecto_repos: [Core.Repo, Core.ReadOnlyRepo, Core.DeduplicationRepo],
```
Comment (PavelVesnin) [inline apps/core/config/config.exs]: in EHealth it's already named ReadRepo. For consistency reasons let's rename it to `Core.ReadRepo`

File: apps/deduplication/lib/deduplication/worker.ex [hunk 2]
```diff
@@ -0,0 +1,50 @@
+defmodule Deduplication.Worker do
+  @moduledoc """
+  Sunchroniously get persons, candidates from replica and mark last updated_at an d locks ids
```
Comment (PavelVesnin) [inline apps/deduplication/lib/deduplication/worker.ex]:
```suggestion
  Synchronously get persons, candidates from replica, mark last updated_at and locks ids
```

File: apps/deduplication/test/v2/genstage_test.exs [hunk 3]
```diff
@@ -13,12 +13,16 @@ defmodule Deduplication.V2.GenStageTest do
...
   alias Deduplication.V2.Model
+  alias Deduplication.Worker
 
   setup :verify_on_exit!
   setup :set_mox_global
 
   setup do
     GenStage.start_link(Producer, %{offset: 0}, name: Producer)
+    :ok = Supervisor.terminate_child(DeduplicationGenStageSupervisor, Worker)
+    # :ok = Supervisor.delete_child(DeduplicationGenStageSupervisor, Worker)
```
Comment (PavelVesnin) [inline apps/deduplication/test/v2/genstage_test.exs]: remove comment

File: apps/person_updates_producer/worker.ex [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    config = Confex.fetch_env!(:mpi_scheduler, __MODULE__)
+
+    __MODULE__.new_job()
+    |> Job.set_overlap(false)
+    |> Job.set_schedule(Parser.parse!(config[config_name]))
+    |> Job.set_task(fun)
+    |> Job.set_run_strategy(%Local{})
+    |> __MODULE__.add_job()
+  end
+end
```
Comment (PavelVesnin) [inline apps/person_updates_producer/worker.ex]: What this file is doing here? 
It has MPIScheduler prefix, don't located in `/lib` folder and not used in code

File: apps/deduplication/lib/application.ex [hunk 5]
```diff
@@ -7,6 +7,7 @@ defmodule Deduplication.Application do
   alias Deduplication.Consumer
   alias Deduplication.Producer
   alias Deduplication.V2.PythonWorker
+  alias Deduplication.Worker
```
Comment (PavelVesnin) [inline apps/deduplication/lib/application.ex]: PR is for ReadRepo, but reworked Deduplication worker.
It should be in different PR

File: apps/core/config/config.exs (hunk 1 above)
Comment (marinakr) [reply apps/core/config/config.exs]: ReadOnly

File: apps/deduplication/lib/application.ex (hunk 5 above)
Comment (marinakr) [reply apps/deduplication/lib/application.ex]: This is part of #5399

File: apps/deduplication/lib/deduplication/worker.ex [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+    procude_and_state(persons, demand, offset, mode)
+  end
+
+  # The order of patterns is nessesary, first check does locked_persons is not empty
+  defp procude_and_state([], _demand, 0, :locked), do: {:reply, [], %{}}
+  defp procude_and_state([], demand, 0, :mixed), do: {:reply, get_persons(demand), %{mode: :new}}
+  defp procude_and_state([], demand, _offset, mode), do: procude_and_state(demand, 0, mode)
+  defp procude_and_state(persons, demand, offset, mode), do: {:reply, persons, %{mode: mode, offset: offset + demand}}
+
+  defp refersh_stat, do: SQL.query!(Repo, "VACUUM ANALYZE verifying_ids")
```
Comment (BatrakM) [inline apps/deduplication/lib/deduplication/worker.ex]: refresh

File: apps/deduplication/config/config.exs [hunk 7]
```diff
@@ -7,6 +7,14 @@ config :logger, :console,
   format: "$message\n",
   metadata: [:request_id]
 
+# ENV DEDUPLICATION_MODE define how producer will get persons
```
Comment (BatrakM) [inline apps/deduplication/config/config.exs]: defines
